src/Parameters.h

```diff
@@ -15,7 +15,7 @@ class Parameters
   explicit Parameters(Configuration const & config)
   {
     params[PAR_NUM] = 20;                     //  0 num params = 20
-    params[PAR_MANU] = MANU_VLCB;              //  1 manf = MERG, 165
+    params[PAR_MANU] = MANU_MERG;              //  1 manf = MERG, 165
```

Comment:
This is for testing. Please don't commit.

Reply:
Reverted to previous version of file